src/main/resources/story/shop_api/GetCategories.story

```diff
@@ -0,0 +1,30 @@
+Story: GET /categories/ test
+
+Scenario: check response body
+When I issue a HTTP GET request for a resource with the URL 'https://api.predic8.de/shop/categories/'
+Then the response code is equal to '200'
+When I save response body to the scenario variable 'body'
+Then a JSON element from '${body}' by the JSON path '$' is equal to '{
+"categories": [
```

Comment:
Can I read somehow expected json from file? It's not convenient to store it inside story as it's bulky

Reply:
How can I check whether actual json matches expected json schema?